lib/themes/dosomething/paraneue_dosomething/js/components/SlideshowController.js

```diff
@@ -1,14 +1,26 @@
+import { sendEvent } from '../utilities/Analytics';
+
 const React = require('react');
 
 /**
  * SlideshowController Component
  * <SlideshowController />
  */
 class SlideshowController extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.finish = this.finish.bind(this);
+  }
+
+  finish() {
```

Comment:
Not something we need to do for this PR, but let's start thinking about naming conventions for methods like this. Maybe `onClose` for the prop (to indicate it's a function prop), and `handleClose` for event handler. Otherwise it feels like a bit of a hunt to figure out how things are wired up.

/cc @weerd